examples/ReactNativeBlobUtil/NuGet.config

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <clear />
+    <add key="react-native" value="https://pkgs.dev.azure.com/ms/react-native/_packaging/react-native-public/nuget/v3/index.json" />
```

Comment:
I dont think we need the react-native feed here anymore. 
We should update react-native-windows template to not include it.
See: https://github.com/microsoft/react-native-windows/issues/14030